product_pricelist_supplierinfo_coefficient/views/product_pricelist_item_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="product_pricelist_item_form_view_inherit" model="ir.ui.view">
```

Comment:
Could you also add `coefficient` column to the items tree view (`product.product_pricelist_item_tree_view`)? This will enable us to show automatically this value also for e.g. `product_category_pricelist_button` without any changes there.
cc @lmiguens-solvos